src/main/java/org/geomerty/procedural/shapes/Triangle.java

```diff
@@ -0,0 +1,7 @@
+package org.geomerty.procedural.shapes;
+
+public class Triangle {
+    public Point topLeft;
+    public double height;
+    public double base;
```

Comment:
@Alexia-GROSS Ah, nice it's one of the ways to represent a triangle. e.g. another one could be 2 sides and angle between them. Here we can see another benefit of encapsulating this details in OOP version.